bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/LedgerEntries.java

```diff
@@ -42,10 +42,15 @@
      * Calling this method does not modify the reference count of the ByteBuf in the returned LedgerEntry objects.
      * The caller who calls {@link #iterator()} should make sure that they do not call ByteBuf.release() on the
      * LedgerEntry objects to avoid a double free.
-     * All reference counts will be decremented when the containing LedgerEntries object is closed.
+     * All reference counts will be decremented when the containing LedgerEntries object is closed via {@link #close()}.
      *
      * @return an iterator of LedgerEntry objects
      */
     @Override
     Iterator<LedgerEntry> iterator();
+
+    /**
+     * Close to release the resources held by this instance.
+     */
+    void close();
```

Comment:
@Override

Reply:
haha, there's a github user called override. poor person must have a mailbox full of spam.

Reply:
empty repo though :)